source/manual/rules-for-getting-production-access.html.md

```diff
@@ -0,0 +1,53 @@
+---
+owner_slack: "#govuk-2ndline"
+title: Rules for getting production access
+parent: "/manual.html"
+layout: manual_layout
+section: 2nd line
+type: learn
+last_reviewed_on: 2019-10-23
+review_in: 3 months
+---
+
+These rules apply to developers in the GOV.UK programme and SREs in the TechOps programme.
+
+## What production access means
+
+- Permission to [read & write production and staging hieradata](https://github.com/alphagov/govuk-secrets/blob/master/puppet/hieradata/production_credentials.yaml) in govuk-secrets using GPG
+- Permission to read & write from the Pass in govuk-secrets store using [GPG](https://github.com/alphagov/govuk-secrets/blob/master/pass/2ndline/.gpg-id)
+- Access to Production Deploy Jenkins and Staging Deploy Jenkins to deploy applications via the [GOV.UK Production GitHub team](https://github.com/orgs/alphagov/teams/gov-uk-production)
+- SSH access to production and staging servers via [govuk-puppet](https://github.com/alphagov/govuk-puppet)
+- AWS [PowerUser Access](https://github.com/alphagov/govuk-aws-data/blob/master/data/infra-security/production/common.tfvars) via the `role_poweruser_user_arns` role
+- Signon "Super Admin" access in production
+
+## When you get production access
+
+- Temporary supervised access during two 2nd line shadow shifts (GOV.UK developers only)
+- Supervised access after second shadow shift and probation has been passed (probation condition does not apply to SREs in TechOps)
+- Permanent access once a non-shadow 2nd line shift has been completed
+
+The above is caveated by a minimum of SC (yellow building pass) security
+clearance, or an application in-process for a suitable level of clearance. This
+in line with [GDS’ general policy that all staff have SC
```

Comment:
There's a smart-apostrophe which should be replaced with a regular single quote.

Reply:
Argh I thought I got them all. Thanks.